frontend/src/pages/login/forgotPassword.tsx

```diff
@@ -0,0 +1,160 @@
+import Auth from '@aws-amplify/auth'
+import {
+  IonButton,
+  IonCard,
+  IonCardContent,
+  IonContent,
+  IonHeader,
+  IonInput,
+  IonItem,
+  IonLabel,
+  IonPage,
+  IonTitle,
+  IonToolbar,
+  useIonToast,
+} from '@ionic/react'
+import React, { useState } from 'react'
+import './login.css'
+
+const ForgotPasswordPage: React.FC = (props: any) => {
+  const [mail, setMail] = useState('')
+  const [codeSent, setCodeSent] = useState(false)
+  const [confirmationCode, setConfirmationCode] = useState('')
+  const [newPassword, setNewPassword] = useState('')
+  const [presentToast] = useIonToast()
+
+  if (props.authState !== 'forgotPassword') {
+    return null
+  }
+
+  const showError = (message?: string) => {
+    presentToast(
+      `Fehler: ${
+        message ||
+        'Etwas ist schiefgelaufen, bitte versuchen Sie es später noch einmal!'
+      }`,
+      2000,
+    )
+  }
+
+  const sendCode = () => {
+    Auth.forgotPassword(mail)
+      .then(data => {
+        console.log('data: ', data)
+        setCodeSent(true)
+      })
+      .catch(error => showError(error?.message))
+  }
+
+  const confirmNewPassword = () => {
+    Auth.forgotPasswordSubmit(mail, confirmationCode, newPassword)
+      .then(_ => {
+        props.onStateChange('signedIn')
+      })
+      .catch(error => showError(error?.message))
+  }
+
+  const backToSignIn = () => {
+    props.onStateChange('signIn')
+  }
+
+  return (
+    <IonPage>
+      <IonHeader>
+        <IonToolbar>
+          <IonTitle>Metapholio</IonTitle>
+        </IonToolbar>
+      </IonHeader>
+      <IonContent>
+        <IonHeader collapse="condense">
+          <IonToolbar>
+            <IonTitle size="large">Metapholio</IonTitle>
+          </IonToolbar>
```

Comment:
Since #37 is merged you can already use the `Header` and `LargeHeader` component